metadata.yml

```diff
@@ -0,0 +1,4 @@
+description:
+  SAP Netweaver deployment formula
+group:
```

Comment:
Maybe we should have the same group for `HANA` and `Netweaver`. What do you think?
Something like `SAP deployment`

Reply:
yes its a good idea I will update it